.circleci/verdaccio/config.yml

```diff
@@ -0,0 +1,27 @@
+storage: ./storage
+auth:
+  htpasswd:
+    file: ./htpasswd
+uplinks:
+  npmjs:
+    url: https://registry.yarnpkg.com/
+    max_fails: 40
+    maxage: 30m
+    timeout: 60s
+    fail_timeout: 10m
+    cache: false
+    agent_options:
+      keepAlive: true
+      maxSockets: 40
+      maxFreeSockets: 10
+packages:
```

Comment:
I'd recommend optimisations like https://github.com/facebook/docusaurus/pull/7967 to avoid the registry proxy private packages to outside (publish might fails due network conditions) also help to keep isolated the packages.

Reply:
This is perfect, thank you so much 🤞